### PR TITLE
Add non-nil check before logging volume errors.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Fixes daemon panic in #31903

Signed-off-by: Anusha Ragunathan <anusha.ragunathan@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

**Comment by vieux** [comment]
LGTM

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: volume/store/store.go [hunk 1]
```diff
@@ -542,7 +542,11 @@ func lookupVolume(driverName, volumeName string) (volume.Volume, error) {
 	if err != nil {
 		err = errors.Cause(err)
 		if _, ok := err.(net.Error); ok {
-			return nil, errors.Wrapf(err, "error while checking if volume %q exists in driver %q", v.Name(), v.DriverName())
+			if v != nil {
+				volumeName = v.Name()
+				driverName = v.DriverName()
```
Comment (AkihiroSuda) [inline volume/store/store.go]: Can we reuse the original driverName in the arg?

Comment (cpuguy83) [inline volume/store/store.go]: This just makes sure to the name is the "real" name.

Comment (cpuguy83) [inline volume/store/store.go]: But I think `v` will always be nil here, so we should just drop this.

Comment (vdemeester) [inline volume/store/store.go]: Let's do a follow-up for that 👼

**Pull request merged**